### PR TITLE
DSM-PEPPER-710-Participant-Withdrawal-Page-UI-update

### PR DESCRIPTION
**Pull request by GiCharkviani**
[PEPPER-710](https://broadworkbench.atlassian.net/browse/PEPPER-710)

Updates:
1. commented out the checkbox as suggested
2. updated the text.

Screenshot 2023-04-05 at 10 40 09

[PEPPER-710]: https://broadworkbench.atlassian.net/browse/PEPPER-710?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html [hunk 1]
```diff
@@ -21,18 +21,21 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
       <mat-form-field>
         <input matInput type="text" [(ngModel)]="participantId" placeholder="Participant ID" maxlength="200" style="width: 10cm">
       </mat-form-field>
-      <mat-checkbox color="primary" disableRipple
-                   [checked]="deletedFromDDP" (change)="deletedFromDDP = $event.checked">Delete from DDP</mat-checkbox>
+<!--      <mat-checkbox color="primary" disableRipple-->
```
Comment (otchet-broad) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html:24]: Do you typically leave these items in code, commented out?

File: ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html [hunk 2]
```diff
@@ -21,18 +21,21 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
...
-        Selecting <u>'Delete from DDP'</u> will not delete the participant data from the DDP (yet), but will prevent DSM from asking the DDP for the Short ID of that participant.
-        <br/>Please select it, if you going to delete the participant from the DDP
-      </b>
+      <p><b>To withdraw a Participant from this study:</b></p>
+      <ol class="instructions_list">
+        <li>Enter their Participant ID above (Note: This is their 20 character Internal Participant ID found on their
+          individual participant page, not their 6 character Short ID)</li>
+        <li>Click the "Withdraw Participant" button</li>
+      </ol>
+      <p><b>Be cautious when using this feature!</b> Once a Participant is withdrawn from a study, it cannot be undone.
```
Comment (otchet-broad) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html:37]:
```suggestion
      <p><b>Be cautious when using this feature!</b> Once a participant is withdrawn from a study, it cannot be undone.
```

File: ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html (hunk 1 above)
Comment (GiCharkviani) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html:24]: Well, usually no, but it was suggested in the ticket

File: ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html (hunk 2 above)
Comment (GiCharkviani) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html:37]: I just copied the same text :D

**Review by otchet-broad — APPROVED** [review]

File: ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html (hunk 1 above)
Comment (pegahtah) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/participant-exit/participant-exit.component.html:24]: This might come back so let's keep the code

**Review by pegahtah — APPROVED** [review]

**Pull request merged**